webapp/channels/src/components/post_edit_history/post_edit_history.scss

```diff
@@ -1,4 +1,4 @@
-.sidebar-right__edit-post-history .scrollbar--view {
+.sidebar-right__edit-post-history .simplebar-content {
```

Comment:
I love that we are putting the new scrollbars in a new component with minimum interface. But it seems we are bleeding some library specifics into this file. Can we avoid it?

(I say it because `simplebar` sounds like the new library :P )

Reply:
I tried to do that wherever I could, but our CSS is so awkward in some of these areas that I couldn't always do that. I would've liked to remove that entirely similar to the `[data-simplebar]` styles I added to the new component, but there was some reason that it didn't work here